src/js/Alert.js

```diff
@@ -45,7 +45,11 @@ class Alert extends React.Component {
         this.setState(prevState => ({ speakCounter: prevState.speakCounter > 0 ? prevState.speakCounter - 1 : '' }))
     }
     componentDidMount() {
-        this.setState({alertCounter: store.getState().ui[store.getState().activeApp].alert.duration/1000});
+        const { activeApp } = store.getState()
+
+        if (activeApp) {
+            this.setState({alertCounter: store.getState().ui[store.getState().activeApp].alert.duration/1000});
```

Comment:
Could it be replaced with suggestion below?
```suggestion
            this.setState({alertCounter: activeApp.alert.duration/1000});
```